doc/source/whatsnew/v0.23.0.txt

```diff
@@ -646,6 +646,7 @@ Groupby/Resample/Rolling
 - Bug in :func:`DataFrame.groupby` where tuples were interpreted as lists of keys rather than as keys (:issue:`17979`, :issue:`18249`)
 - Bug in :func:`DataFrame.transform` where particular aggregation functions were being incorrectly cast to match the dtype(s) of the grouped data (:issue:`19200`)
 - Bug in :func:`DataFrame.groupby` passing the `on=` kwarg, and subsequently using ``.apply()`` (:issue:`17813`)
+- Bug in :func:`DataFrame.aggregate` on non-existant columns (:issue:`16766`)
```

Comment:
Say what the issue was, also I think the issue was with `DataFrame.resample().aggregate`, not `DataFrame.aggregate`, right?

So something like

    Bug in :func:`DataFrame.resample` not raising a a `KeyError` when aggregating a non-existent column.